addons/website/static/src/js/content/menu.js

```diff
@@ -531,8 +536,12 @@ publicWidget.registry.menuDirection = publicWidget.Widget.extend({
      * @private
      */
     _onDropdownShow: function (ev) {
-        var $li = $(ev.target);
+        // $li is a div if the dropdown is the o_extra_menu_items one.
+        const $li = $(ev.target).closest('.nav-item.dropdown');
```

Comment:
Since it's not always a `<li>` shouldn't we change the name of this variable? @qsm-odoo Do you think it is safein V16 ?

Reply:
There are 20 lines in this function, I don't think making a variable name clearer will generate too many diff (since you have to touch some of those lines anyway) :wink: But yes, case by case, if it was 200 lines, maybe reserve this change to a master refactoring... but here it is seems to be about 3-4 lines so let's go for it I'd say :grimacing: